test/integration/targets/ansible-galaxy/runme.sh

```diff
@@ -2,6 +2,14 @@
 
 set -eux -o pipefail
 
+# Prep the local git repos with role and make a tar archive so we can test
+# different things
+galaxy_local_test_role="test-role"
+galaxy_local_test_role_dir=$(mktemp -d)
```

Comment:
I'd use a directory under `$OUTPUT_DIR` instead of `mktemp` here and other places in the test.

Reply:
Fixed, and I've amended all other uses of mktemp in the script (including unrelated tests)